src/main/resources/com/google/api/codegen/java/PUBLISHING.md

```diff
@@ -0,0 +1,31 @@
+PUBLISHING
+----------
+
+Note: This folder has been generated by the gen-api-package tool.
+
+The instructions assume that no changes have been made to the folder and its
+contents since it was created.
+
+PREREQUISITES
+-------------
+
+- JDK 7 must be installed.
```

Comment:
Actually, JDK 6, according to the values in the build.gradle:

```
  sourceCompatibility = 1.6
  targetCompatibility = 1.6
```

It should probably say "JDK 6 or later must be installed."